generic/tests/netperf.py

```diff
@@ -118,10 +119,62 @@ def mtu_set(mtu):
         if params.get("reboot_after_config", "yes") == "yes":
             vm.reboot(method="system_reset", serial=True)
 
-    try:
-        vm.wait_for_serial_login(timeout=login_timeout, restart_network=True).close()
-    except virt_vm.VMIPAddressMissingError:
-        pass
+    if params.get("failover_pair_id_nic1"):
+        if params.get("os_type") == "linux":
+            session = vm.wait_for_serial_login(timeout=login_timeout)
+            ifname = utils_net.get_linux_ifname(session)
+            for i in ifname:
+                cmd = "ethtool -i %s |grep driver| awk -F': ' '{print $2}'" % i
+                driver = session.cmd_output(cmd).strip()
+                if driver == "net_failover":
+                    session.cmd_output("dhclient -r && dhclient %s" % i)
+                break
+        if params.get("os_type") == "windows" and params.get("install_vioprot_cmd"):
+            media_type = params.get("virtio_win_media_type")
```

Comment:
It should be params["virtio_win_media_type"] here?

Reply:
updated.